core/scripts/testing/scripts.js

```diff
@@ -47,6 +47,11 @@
       linkTag.setAttribute('href', '/css/ionic/bundle.ionic.css');
       document.head.appendChild(linkTag);
     }
+
+    const defaultThemeLinkTag = document.querySelector('link[href*="css/ionic.bundle.css"]');
```

Comment:
@brandyscarney I think this file doesn't exist 🤔, or am I missing something? Maybe that's the problem, and the link on the tests pages need to be updated or removed? 

At least for Ionic Theme tests, all its needed should be the one added on line 47